platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/LogoutMessage.java

```diff
@@ -38,7 +39,10 @@ LogoutResponse extractSamlLogoutResponse(@NotNull String samlLogoutResponse)
   LogoutRequest extractSamlLogoutRequest(@NotNull String samlLogoutRequest)
       throws XMLStreamException, WSSecurityException;
 
-  LogoutRequest buildLogoutRequest(@NotNull String nameIdString, @NotNull String issuerOrEntityId);
+  LogoutRequest buildLogoutRequest(
```

Comment:
❓ Could we remove the `@NotNull` in this file and use `@Nullable` where appropriate instead to follow our convention?